src/instructlab/model/chat.py

```diff
@@ -464,7 +464,9 @@ def _handle_context(self, content):
             )
             raise KeyboardInterrupt
         self.loaded["name"] = context
-        self.loaded["messages"] = [{"role": "system", "content": CONTEXTS[context]}]
+
+        sys_prompt = CONTEXTS.get(context, "default")(get_model_arch(self.model))
```

Comment:
is this syntax right?

Reply:
yes, i tested it

Reply:
you can test it as well - if you run `ilab model chat` on any GGUF model and run `/c context default` or just ask it "who are you" you should see the generic prompt I added, instead of the one that says " I am a granite 7b model made by Red hat...."